### DISCUSSION TITLE
SPI Controller

### DISCUSSION THREAD

**Discussion by npilar** [eblot/pyftdi#265] [Q&A] [answered]
I am trying use the quickstart example for SPI communication for FT2232 Mini Module.  

spi = SpiController()
NameError: name 'SpiController' is not defined

How would I be able to define SpiController?  What needs to be imported?

**Comment by eblot** [comment, marked as answer]
> How would I be able to define SpiController?  What needs to be imported?
> 

See for example

https://github.com/eblot/pyftdi/blob/e2a4eacad5082feaff4921c8e16691861f6d94c8/pyftdi/tests/spi.py#L22
